_pages/user-experience/cancel-url.md

```diff
@@ -11,13 +11,32 @@ sidenav:
     href: "user-experience/agency-logo/"
   - text: Cancel URL
     href: "user-experience/cancel-url/"
+    links:
+      - text: OIDC Redirect URI
+        href: "#oidc-redirect-uri"
+      - text: SAML Return to SP URL
+        href: "#saml-return-to-sp-url"
   - text: Failure to proof URL
     href: "user-experience/failure-proof/"
   - text: Knowledge articles
     href: "user-experience/knowledge-articles/"
-  - text: FAQ content 
+  - text: FAQ content
     href: "user-experience/faq-content/"
 
 ---
 
-## Cancel URL
+## Determine your application’s Return to service provider URL
```

Comment:
Agree, perhaps if it were in quotes it would fit better, since its the name of the input.

> Determine your application's "Return to service provider URL"

Reply:
I say lower case.

Reply:
what if i put it in backticks?

Reply:
This is what backticks would look like Karen, not sure how familiar you are with markdown.

> Detemine your application's `Return to service provider URL`

Reply:
I would rather not have backticks in the header, but if it is a matter of accuracy then I guess that is alright.

Reply:
i updated it to lower case in https://github.com/18F/identity-dev-docs/pull/417/commits/f6212ed9e56021d7e0701d26ab1d7869cacf2a46. that was what was in the draft anyways. we can always update it again if it's confusing for folks! thanks for workshopping with me